src/components/apps/Notes/Notes.svelte

```diff
@@ -0,0 +1,145 @@
+<script>
```

Comment:
Please make this `lang="ts"`. This project strictly uses TypeScript to maintain code quality